docs/kms_secretlock.md

```diff
@@ -232,4 +232,66 @@ a, err = aries.New(aries.WithKMS(func(ctx kms.Provider) (kms.KeyManager, error)
 }))
 ```
 
-Aries framework will use the instance of customKMS passed in as an option instead of creating a default one.
\ No newline at end of file
+Aries framework will use the instance of customKMS passed in as an option instead of creating a default one.
+
+## Interop with external keys
+
+### Export Public signing keys []bytes from KMS
+The default KMS implementation (in `localkms` package) supports exporting public signing keys in DER/IEEE-P1363 signature formats to allow signature verification outside of the KMS.
+
+The following call is an example of creating and exporting a public P-256 key using IEEE-P1363 signature format:
+
+```
+// ... get/create kmsInstance as per previous sections
+keyID, newKeyHandle, err := kmsInstance.Create(kms.ECDSAP256TypeIEEEP1363)
+if err != nil {
+    return err
+}
+
+pubKeyBytes, err := kmsInstance.ExportPubKeyBytes(ksID)
+
+```
+
+### Convert Public signing keys []byte to Tink's keyset.Handle
+
+The reverse of an export operation would be to get a keyset.Handle from a public key in raw []byte (DER or IEEE-P1363 signing format). The KMS service *does not* support importing *public keys*, but it can create a Tink `keyset.Handle` instance representing a public signing key to execute signature verification.
```

Comment:
> The KMS service *does not* support importing *public keys*

I assume this means only by themselves? i.e., there is private key import below, so I assume the public keys come along in that case?

Reply:
correct, importing private key includes its public key

Reply:
I've updated the readme to clarify this this detail.